_sass/youth-managed-farms.css

```diff
@@ -0,0 +1,53 @@
+* {
+  box-sizing: border-box;
+}
+
+html, body {
+  margin: 0;
+  padding: 0;
+  background-color: #272b34;
+  color: #fff;
+}
+.box-content {
```

Comment:
We may need to add padding to the `top` and `bottom` instead of margin to include the background